fields/class-cmb-number-field.php

```diff
@@ -32,7 +34,7 @@ public function default_args() {
 	public function html() {
 		?>
 
-		<input step="<?php echo esc_attr( $this->args['step'] ); ?>" type="number" <?php $this->id_attr(); ?> <?php $this->boolean_attr(); ?> <?php $this->class_attr( 'cmb_text_number code' ); ?> <?php $this->name_attr(); ?> value="<?php echo esc_attr( $this->get_value() ); ?>" />
+		<input step="<?php echo esc_attr( $this->args['step'] ); ?>" <?php echo '' !== $this->args['min'] ? printf( 'min="%d"', $this->args['min'] ) : ''; ?> <?php echo '' !== $this->args['max'] ? printf( 'max="%d"', $this->args['max'] ) : ''; ?> type="number" <?php $this->id_attr(); ?> <?php $this->boolean_attr(); ?> <?php $this->class_attr( 'cmb_text_number code' ); ?> <?php $this->name_attr(); ?> value="<?php echo esc_attr( $this->get_value() ); ?>" />
```

Comment:
If i was being really picky, printf inside ternary operators i think makes this hard to read, perhaps defaults should be dealt with at the beginning of the the method so this can be just `<?php echo $min_attr; ?>`, is there any precedent for having `<?php echo '' !== $this->args['min'] ? printf( 'min="%d"', $this->args['min'] ) : ''; ?>` within the markup output line within other fields of this plugin?